### PR TITLE
Merge AlertIOS with Alert 

### PR DESCRIPTION
**Pull request by wellmonge**
## Summary

Itwas merged AlertIOS into Alert and removed type parameter from Alert.alert line 60 at Alert.js 
 
## Changelog
[AlertIOS] [Change and Replace] - Merge AlertIOS into Alert.

## Test Plan

CI is green, and everything should work as normal.

### CODE REVIEW

File: ReactAndroid/src/main/java/com/facebook/react/modules/debug/interfaces/DeveloperSettings.java [hunk 1]
```diff
@@ -1,4 +1,3 @@
-/**
```
Comment (cpojer) [inline ReactAndroid/src/main/java/com/facebook/react/modules/debug/interfaces/DeveloperSettings.java]: Can you revert this change?

File: Libraries/react-native/react-native-implementation.js [hunk 2]
```diff
@@ -172,9 +172,6 @@ module.exports = {
   get Alert() {
     return require('Alert');
   },
-  get AlertIOS() {
```
Comment (cpojer) [inline Libraries/react-native/react-native-implementation.js]: This is used in other places. Can you search for `AlertIOS` in the repository and remove uses of this module and update tests to use `Alert` instead?

Here, it might make sense to actually throw and tell users to use `Alert` instead.

File: Libraries/Alert/Alert.js [hunk 3]
```diff
@@ -10,56 +10,151 @@
...
+ */
+type AlertType = $Enum<{
+    default: string,
+    'plain-text': string,
+    'secure-text': string,
+    'login-password': string,
+  }>;
+  
+  /**
+   * An Alert button style
```
[the remark this answers is not in the document]
Comment (cpojer) [reply Libraries/Alert/Alert.js]: Seems like prettier didn't run properly. Can you run `yarn` and then `yarn lint --fix`?

**Comment by wellmonge** [comment]
> Seems like there is gonna be a bit more work necessary to pull this

File: Libraries/react-native/react-native-implementation.js (hunk 2 above)
Comment (wellmonge) [reply Libraries/react-native/react-native-implementation.js]: Okay, I'll do that.

File: Libraries/Alert/Alert.js [hunk 4]
```diff
@@ -10,13 +10,19 @@
...
-const AlertIOS = require('AlertIOS');
 const NativeModules = require('NativeModules');
+const RCTAlertManager = NativeModules.AlertManager;
 const Platform = require('Platform');
 
-import type {AlertType, AlertButtonStyle} from 'AlertIOS';
+/**
+ * Array or buttons
+ * @typedef {Array} Buttons
+ * g@property {string=} text Button label
```
Comment (guhungry) [inline Libraries/Alert/Alert.js]:
```suggestion
 * @property {string} text Button label
```

**Comment by wellmonge** [comment]
@guhungry, @cpojer could anyone help with those failure at CI js_coverage?

**Comment by guhungry** [comment]
> @guhungry, @cpojer could anyone help with those failure at CI js_coverage?

Why it was run on different circle account
https://circleci.com/gh/wellmonge/react-native/118?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
vs where it should be
https://circleci.com/gh/facebook/react-native/tree/pull%2F23318

**Comment by guhungry** [comment]
@wellmonge If you can trigger build for 4b53dd914ff4455bb372ec61e0b92ce257f03d57 at https://circleci.com/gh/facebook/react-native/tree/pull%2F23318. I think all ci will be pass.

**Comment by wellmonge** [comment]
> @wellmonge If you can trigger build for [4b53dd9](https://github.com/facebook/react-native/commit/4b53dd914ff4455bb372ec61e0b92ce257f03d57) at https://circleci.com/gh/facebook/react-native/tree/pull%2F23318. I think all ci will be pass.

Okay, later I'll do that

**Comment by wellmonge** [comment]
> The code looks good! I reverted a few of the unintentional changes but we should be able to land this at FB soon.

Okay, let me know if had to do some other changes.
